Update Lighthouse TTFB documentation to Server Response Times (#6455)
GoogleChrome/developer.chrome.com: site/en/docs/lighthouse/performance/server-response-time/index.md

Users dislike when pages take a long time to load.
Slow server response times are one possible cause for long page loads.

{% Aside 'important' %}

> Lighthouse usually measures _Server Response Time_, rather than the full [Time to First Byte (TTFB)](https://web.dev/ttfb/). As well as including the Server Response Time, TTFB can often include DNS lookups and redirects (for example, if the final slash or the www subdomain or https protocol is omitted then the server might redirect to the correct URL, or for URL shorteners or Ads that redirect via several domains). Most Lighthouse tools will test the end URL only, missing the DNS and redirect parts.

Reviewer: possible change:
```suggestion
Server Response Time is only part of the full [Time to First Byte (TTFB)](https://web.dev/ttfb/). As well as including the Server Response Time, TTFB can often include DNS lookups and redirects (for example, if the final slash or the www subdomain or https protocol is omitted then the server might redirect to the correct URL, or for URL shorteners or Ads that redirect via several domains). Most Lighthouse tools will test the end URL only, missing the DNS and redirect parts.
```

Current wording seems a little general considering this is the docs for the server response time audit :)
